lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -433,7 +434,7 @@ def _get_job_reqs_from_concierge_params(
             cpus=norm.get(REQUEST_CPUS),
             memory_MB=norm.get(REQUEST_MEMORY),
             disk_GB=norm.get(REQUEST_DISK),
-            client_group=norm.get(CLIENT_GROUP),
+            client_group=norm.get(CLIENT_GROUP) or CONCIERGE_CLIENTGROUP,
```

Comment:
Interesting.. I've always seen it done as using the default from the get, versus using an "or"
```
client_group=norm.get(CLIENT_GROUP,CONCIERGE_CLIENTGROUP)
```

Reply:
Heh, yeah, I guess my head just went to the `or` as it doesn't change what's already there. They're completely equivalent AFAICT - it actually makes me wonder why the 2 argument form of `get` is necessary.